protoutil/conversion.go

```diff
@@ -39,18 +40,13 @@ func ToProtoStruct(v any) (*structpb.Struct, error) {
 	return x, nil
 }
 
-// UnmarshalYAML unmarshals the give YAML bytes to the given proto message.
-func UnmarshalYAML(b []byte, msg proto.Message) error {
-	tmp := map[string]any{}
-	if err := yaml.Unmarshal(b, tmp); err != nil {
-		return fmt.Errorf("failed to unmarshal yaml: %w", err)
-	}
-	jb, err := json.Marshal(tmp)
+// YAMLToProto converts YAML to Proto.
+func YAMLToProto(y []byte, m proto.Message) error {
```

Comment:
This is now `protoutil.YAMLToProto`. I wonder if `protoutil.FromYAML` might be easier?

Reply:
Yea, pretty nit but in general I'd like to avoid duplicated info that already presents in the package name.

Reply:
done